client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/MlClientDocumentationIT.java

```diff
@@ -1789,7 +1798,10 @@ public void testPostData() throws Exception {
             postDataRequest.setResetStart(null);
 
             // tag::post-data-execute
-            PostDataResponse postDataResponse = client.machineLearning().postData(postDataRequest, RequestOptions.DEFAULT);
+            RequestOptions ignoreDeprecationsWarningsOptions = RequestOptions.DEFAULT.toBuilder()
```

Comment:
Just double-checking: Is it ok that this snippet will be printed in the public docs?

Reply:
This is the worst thing about this PR.  The end user won't need to ignore the deprecation warnings, because they won't be fatal for the end user, only our test framework.  But I tried a few ways to avoid this while still keeping the code executable and they didn't work - putting it in an `if (false)` with the `else` doing the version that works in the test framework and putting the appropriate end user code in a `try`/`catch (AssertionError e)`.

I just thought of a couple of other ways to do it that might work - putting the tagged code in a `/*`/`*/` block comment or in a dummy method that's not called.  I'll see if those work.

Reply:
> putting the tagged code in a `/*`/`*/` block comment

This approach passed the PR CI.  I don't know what it will look like in the final HTML docs though.  I will check after this is merged and if it doesn't work then try a different approach in a followup.